_contentTemplates/gantt/state.md

```diff
@@ -0,0 +1,608 @@
+#set-sort-from-code
+@*Set sorting from code through the Gantt state*@
+
+@using Telerik.DataSource
+
+<TelerikButton ThemeColor="primary" OnClick="@SetGanttSort">Set sort from code</TelerikButton>
```

Comment:
"primary" looks like a magic word. We can either use the `ThemeConstants`, or even better, drop the parameter, as it's not important here and pollutes the example.